package.json

```diff
@@ -96,12 +96,11 @@
     "raf": "^3.4.0",
     "react": "^16.4.2",
     "react-dom": "^16.4.2",
-    "react-intl": "^2.7.2",
+    "react-intl": "^2.8.0",
     "react-router-dom": "^4.2.2",
     "react-test-renderer": "^16.4.2",
     "stylelint": "^9.2.0",
     "stylelint-config-terra": "^2.0.0",
-    "terra-base": "^4.0.0",
```

Comment:
With removing terra-base from the root package.json, it will no come from terra-devs-site.

```
terra-framework > terra-dev-site > terra-base@4.4.0
```